composer.json

```diff
@@ -28,6 +28,7 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "ext-json": "*",
+        "ext-zip": "*",
```

Comment:
Understand why, but this seems kinda big just for patch update?

Reply:
Fixed: https://github.com/ibexa/admin-ui/pull/1271/commits/dc48c6311d682b57b679c9cfe0ee69cdee7a05f5